### PR TITLE
Memfix for doom and shoot

### PR DESCRIPTION
**Pull request by mrsteyk**
omfg its fucking dirty and i want kms so hard. Also did a little format.

### CODE REVIEW

**Comment by mrsteyk** [comment]
Lmfao. Fucking cloack BS is getting me nervous, but it should be ok since you have freaking doom on secondary attack.

File: src/hacks/LagExploit.cpp [hunk 1]
```diff
@@ -323,53 +323,62 @@ void CreateMove()
...
-                      (g_pUserCmd->buttons & IN_ATTACK2);
+            bWasHolding = (g_pUserCmd->buttons & IN_ATTACK) ||
+                          (g_pUserCmd->buttons & IN_ATTACK2);
+        }
     }
     else if (!cloak)
     {
         if (shoot)
         {
-            if (not g_pLocalPlayer->holding_sniper_rifle)
```
Comment (BenCat07) [inline src/hacks/LagExploit.cpp]: just replace all ! with not instead tf

Comment (mrsteyk) [reply src/hacks/LagExploit.cpp]: b4d

**Comment by mrsteyk** [comment]
minigun works strange. ToDo: try add extra button checks for shoot. :joy:

**Comment by mrsteyk** [comment]
should be merged ideally

**Comment by mrsteyk** [comment]
@BenCat07 is lazy. (sadly)

**Pull request closed without merging**